resources/assets/js/app/views/votes/ShowVote.js

```diff
@@ -13,13 +19,57 @@ module.exports = Marionette.LayoutView.extend({
         answers: '#answers'
     },
     ui: {
-        c_count: '#count'
+        c_count: '#count',
+        newCommentButton: '.new-comment-notification'
     },
+
+    events: {
+        'click @ui.newCommentButton': 'showNewComments'
+    },
+
+
+    initialize: function () {
+        this.listenTo(Radio.channel('votesChannel'), 'setCommentsCount' + this.options.voteModel.id, function (n) {
+            this.ui.c_count.text(n + ' Comments');
+        });
+
+        socketCommentClient.bind('VoteComments', this.options.voteModel.id);
+    },
+
+    onBeforeDestroy: function () {
+        this.stopListening();
+        socketCommentClient.unbind('VoteComments', this.options.voteModel.id);
+    },
+
+    onShow: function () {
+        this.addedCommentsCollection = new CommentsCollection([], {parentUrl: ''});
+
+        var self = this;
+        this.collection.listenTo(Radio.channel('VoteComments'),
```

Comment:
this.collection.listenTo(Radio.channel('VoteComments'),
            'newComment', function (comment) {

should be:

this.collection.listenTo(
            Radio.channel('VoteComments'),
            'newComment', 
            function (comment) {
....

or 

this.collection.listenTo(Radio.channel('VoteComments'), 'newComment', function (comment) {
.....